com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/utils/MigrationCfgUtils.java

```diff
@@ -938,6 +947,11 @@ public boolean checkMultipleSchema(Catalog catalog, MigrationConfiguration cfg)
 				&& (catalog.getSchemas().size() > 1)
 				&& !cfg.hasObjects2Export();
 	}
+	
+	public boolean isSourceMultiSchema(Catalog catalog, MigrationConfiguration cfg) {
```

Comment:
method isSourceMultiSchema is newly defined but not referenced.

Reply:
I forgot to erase it. I'll fix it.